src/Utils/FileSystem.php

```diff
@@ -131,11 +131,14 @@ public static function read(string $file): string
 
 	/**
 	 * Writes a string to a file.
+	 * @param $mode int|NULL
```

Comment:
Not sure how to write php doc here as you usually do not use @param with $variableName.